openassessment/xblock/openassessmentblock.py

```diff
@@ -66,6 +67,12 @@
         "navigation_text": "Your assessment of your response",
         "title": "Assess Your Response"
     },
+    "staff-assessment": {
+        "name": "staff-assessment",
+        "class_id": "openassessment__staff-assessment",
```

Comment:
I'd been hoping to remove these special ids, but I hadn't noticed before that the Python code had some knowledge of them too.